FlowCryptAppTests/Functionality/Services/ComposeMessageServiceTests.swift

```diff
@@ -373,7 +373,7 @@ class ComposeMessageServiceTests: XCTestCase {
 
     func testSuccessfulMessageValidation() async throws {
         encryptedStorage.getKeypairsResult = [keypair]
-        recipients.enumerated().forEach { element, index in
+        for _ in recipients {
```

Comment:
This looks odd. Not your fault - it highlights that it was originally odd. I think we should review this test in another issue.

Reply:
ok